packaging/nntrainer.spec

```
@@ -59,6 +62,12 @@ Requires:	openblas-devel
 Development pacage for custom nntrainer developers.
```

Not in this PR though...
```suggestion
Development package for custom nntrainer developers.
```